demo/demo.ts

```diff
@@ -110,70 +113,84 @@ export function getCommonParentDirectory(fileNames: string[]): string {
 
   // Read further settings from tsconfig.json.
-  const projectDir = options.project || '.';
-  const configFileName = path.join(projectDir, 'tsconfig.json');
-  const {config: json, error} =
-      ts.readConfigFile(configFileName, path => fs.readFileSync(path, 'utf-8'));
+  const projectDir = options.project || ".";
+  const configFileName = path.join(projectDir, "tsconfig.json");
+  const { config: json, error } = ts.readConfigFile(configFileName, (path) =>
+    fs.readFileSync(path, "utf-8")
+  );
   if (error) {
-    return {options: {}, fileNames: [], errors: [error]};
+    return { options: {}, fileNames: [], errors: [error] };
   }
-  ({options, fileNames, errors} =
-       ts.parseJsonConfigFileContent(json, ts.sys, projectDir, options, configFileName));
+  ({ options, fileNames, errors } = ts.parseJsonConfigFileContent(
+    json,
+    ts.sys,
+    projectDir,
+    options,
+    configFileName
+  ));
   if (errors.length > 0) {
-    return {options: {}, fileNames: [], errors};
+    return { options: {}, fileNames: [], errors };
   }
 
   // if file arguments were given to the typescript transpiler then transpile only those files
   fileNames = tsFileArguments.length > 0 ? tsFileArguments : fileNames;
 
-  return {options, fileNames, errors: []};
+  return { options, fileNames, errors: [] };
 }
 
 /**
  * Compiles TypeScript code into Closure-compiler-ready JS.
  */
 export function toClosureJS(
-    options: ts.CompilerOptions, fileNames: string[], settings: Settings,
-    writeFile: ts.WriteFileCallback): tsickle.EmitResult {
+  options: ts.CompilerOptions,
+  fileNames: string[],
+  settings: Settings,
+  writeFile: ts.WriteFileCallback
+): tsickle.EmitResult {
   // Use absolute paths to determine what files to process since files may be imported using
   // relative or absolute paths
-  const absoluteFileNames = fileNames.map(i => path.resolve(i));
+  const absoluteFileNames = fileNames.map((i) => path.resolve(i));
 
   const compilerHost = ts.createCompilerHost(options);
   const program = ts.createProgram(absoluteFileNames, options, compilerHost);
   const filesToProcess = new Set(absoluteFileNames);
-  const rootModulePath = options.rootDir || getCommonParentDirectory(absoluteFileNames);
+  const rootModulePath =
+    options.rootDir || getCommonParentDirectory(absoluteFileNames);
   const transformerHost: tsickle.TsickleHost = {
     rootDirsRelative: (f: string) => f,
     shouldSkipTsickleProcessing: (fileName: string) => {
       return !filesToProcess.has(path.resolve(fileName));
     },
     shouldIgnoreWarningsForPath: (fileName: string) => !settings.fatalWarnings,
     pathToModuleName: (context, fileName) =>
-        tsickle.pathToModuleName(rootModulePath, context, fileName),
+      tsickle.pathToModuleName(rootModulePath, context, fileName),
     fileNameToModuleId: (fileName) => path.relative(rootModulePath, fileName),
-    googmodule: true,
+    googmodule: false,
+    es6module: true,
```

Comment:
This is setting the new option, the rest of the changes in this file are essentially style changes.